### ISSUE TITLE
Installation Issue/Windows 7

### ISSUE THREAD

**Issue by jenzmarkallen** [CXuesong/OutlookFolderNotifier#1]
When trying to run setup I get the following, Windows 7 Pro, Outlook 2016 
![error_on_install](https://user-images.githubusercontent.com/25798204/36105312-49ef66fc-0fe2-11e8-99b6-958628f1ad15.GIF)

**Comment by CXuesong** [comment]
Well, this is unexpected. You are using Windows 7 with SP1 installed, right? What is your CPU architecture, x86 or x64?

I've tried to search by error code, but what I found is a little bit confusing. Above all, have you tried to reinstall VSTO? You may download it at https://www.microsoft.com/en-us/download/details.aspx?id=48217

**Comment by jenzmarkallen** [comment]
Yes SP1 and 64 bit.  I will try and reinstall VSTO and let you know

**Comment by jenzmarkallen** [comment]
Nope, same error after reinstall of VTSO

**Comment by CXuesong** [comment]
Then you may try to replace the VSTOInstaller.exe in
```
C:\Program Files\Common Files\microsoft shared\VSTO\10.0
```
Attached is the VSTOInstaller.exe of version 10.0.60301 copied from my PC, while the product version on msdownload is 10.0.60724. It's a little bit strange that the VSTO version as provided as a part of VS 2017 is not the latest.

According to Gyula's post in [this question](https://answers.microsoft.com/en-us/windows/forum/windows_10-other_settings/visual-studio-tools-for-office-solution-installer/04e0572a-22f9-4a0e-a172-decdbd289370), perhaps downgrading VSTOInstaller would solve the problem, but I cannot find the installer of version 10.0.60301 anywhere on the Internet.

**Attachment**
* [VSTOInstaller.zip](https://github.com/CXuesong/OutlookFolderNotifier/files/1722544/VSTOInstaller.zip)

**Comment by jenzmarkallen** [comment]
Renamed the existing VSTOInstaller.exe, copied from your zip to same dirrectory listed above, tried your setup, same issue. The file you sent and the one I renamed are EXACTLY the same size in bytes which I found strange if they are 2 different versions
